### ISSUE TITLE
Citator kickoff email

### ISSUE THREAD

**Issue by s-taube** [freelawproject/courtlistener#4494]
Email to the group. Draft [here](https://docs.google.com/document/d/1fsXSjZB_ZQYNUrQR6ezbbDcdncRPplxORkTDC-AViDA/edit).

**Comment by s-taube** [comment]
@mlissner take a look when you have a sec.

**Comment by mlissner** [comment]
Love it, no changes! :shipit: !

**Comment by s-taube** [comment]
@mlissner Ok, I added a calendly link for Oct 1-4, 9am - 2pm PT (2pm being 11pm in Spain). I took your and my availabilities into account.

Do you want to share the list of emails? CC is okay for all?

**Comment by mlissner** [comment]
Oh, sorry, I was imagining you using a doodle or something to get the availability of folks, since there are so many.

**Comment by s-taube** [comment]
Yes, that's what the calendly link is for -- to get all availabilities. The doodle interface looked awkward and ad-heavy so I switched to calendly.

**Comment by mlissner** [comment]
Oh, I get it, sorry, great.
